examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -21,22 +21,34 @@
      "name": "stdout",
```

Comment:
<p>The note on <code>model_logp</code>  has to be updated</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>multiple samplers doesnt have a variable model_logp, so i removed the cell there, and lp is defined for the first sampling case</p>